src/main/kotlin/com/ex_dock/ex_dock/frontend/template_engine/template_data/single_use/SingleUseTemplateData.kt

```diff
@@ -0,0 +1,11 @@
+package com.ex_dock.ex_dock.frontend.template_engine.template_data.single_use
+
+class SingleUseTemplateData{
```

Comment:
Is it an idea to move the constructor from the SingleUseTemplateData to the primary constructor and make the class a data class? Since it only holds data and doesn't do anything else.